source/matrix_free/task_info.cc

```diff
@@ -1014,10 +1014,61 @@ namespace internal
           blocks                      = {0, comm_begin, comm_end, end};
         }
 
-      // Step 7: Fill in the data by batches for the locally owned cells.
+      // Step 7: sort ghost cells according to the category
+      std::vector<std::tuple<unsigned int, unsigned int>>
```

Comment:
Is there a reason to choose `tuple` over `std::array<unsigned int, 2>` as is used in line 968 above?